src/tests/components/post/PostBlock.test.jsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { shallow } from "enzyme";
+import PostBlock from "../../../components/post/PostBlock";
+import demoImage from "../../../images/demo-img.png";
+
+describe("Post Block Components", () => {
+  const content = "test blog content";
+  const user = "testuser";
+  const postTime = "10 hours ago";
```

Comment:
We can just put these straight into the `<PostBlock>` definition below if we aren't reusing them.

Reply:
done